master-qa-21571

patriciajeanperez opened the pull request:
https://issues.liferay.com/browse/LRQA-21571

Commit:
LPS-63855 Don't require tlds that you provide

pat270 committed:
LPS-63907 - Admin Content section should use Lexicon tables in list view

pat270 committed:
LPS-63907 - Polls portlet list view header has wrong classes

pat270 committed:
LPS-63907 - Table min-width styles are covered in Lexicon

dantewang committed:
LPS-63617 More LoggingTimer, super class: BaseDBProcess

dantewang committed:
LPS-63617 Add common method for UpgradeTable

dantewang committed:
LPS-63617 One more try-with-resources

dantewang committed:
LPS-63617 No more need this since the super method itself can log time info

ctampoya committed:
LRQA-22978 Remove Guest viewing permissions

ctampoya committed:
LRQA-22954 Deprecate DDL visual grouping paths and macros

ctampoya committed:
LRQA-22954 Add macro to view nested fields

ctampoya committed:
LRQA-22954 Add macro to view nested repeating fields

ctampoya committed:
LRQA-22954 Update test name and use case

ctampoya committed:
LRQA-22954 Auto SF

ctampoya committed:
LRQA-22994 Declare variable

Commit:
LPS-64052 Remove PHP support

pei-jung committed:
LPS-59671 Add breadcrumbs

pei-jung committed:
LPS-59671 Show the management bar even when there are no elements displayed

Commit:
LRQA-21076 Update configureExportSiteOptions

Commit:
LRQA-21076 Fix ExportImportValidation test case

vicnate5 committed:
LRQA-23025 Add additional navigation

pei-jung committed:
LPS-59671 The style of the remove icon is off

pei-jung committed:
LPS-59671 Do not show the action button in the select users view

Commit:
LPS-64051 - should use image-viewer instaed of gallery, and sepcify z-index

Commit:
LPS-64051 - com.liferay.portlet.header-portlet-css should point to css not scss

Commit:
LPS-63977 - Add type="text/javascript" attribute for Javascript <script /> tags

pei-jung committed:
LPS-59671 Add kebab menu

Commit:
LPS-63977 - Use <aui:script position="inline" />

natecavanaugh committed:
LPS-63907 - Source formatting

pei-jung committed:
LPS-59671 Add dependencies

ctampoya committed:
LRQA-23031 Use correct category name

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/1101/">test-portal-acceptance-pullrequest(master)</a></p>

ctampoya committed:
LRQA-23027 Add Edit portlet actions path

ctampoya committed:
LRQA-23027 Select frame before editing

ctampoya committed:
LRQA-23026 Find 'Impersonate User' on User menu item

vicnate5 committed:
LRQA-23027 Remove unused path

vicnate5 committed:
LRQA-22954 Remove unused images

kenjiheigel committed:
LRQA-23081 Update table xpath locators to account for additional nested elements

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 2 hours 14 minutes 56 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/65d68a32633977f0457545627b27071de7268cb5">65d68a32633977f0457545627b27071de7268cb5</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/1101/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23146/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23147/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26892/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30271/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31302/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31111/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22624/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/35015/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24159/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30274/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22626/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22625/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26651/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22628/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26893/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31059/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/35016/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26654/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/34786/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/24</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26655/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/25</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31807/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/26</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26656/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/27</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/34466/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24158/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23750/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22623/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23151/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30272/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/35013/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31117/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31809/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23752/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk7</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23518/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24160/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23152/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26896/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk7</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22722/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk7</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31304/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31305/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk7</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31306/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk7</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/35017/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26658/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/1303/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-20.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1101/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/1101/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>43 Jobs Passed.<br />1 Job Failed.</p><pre>Completed with the status of FAILURE.</pre></li></ol>

sharonchoi commented:
No test changes needed :)

pei-jung committed:
LPS-59671 Unable to delete user group from kebab menu

dantewang committed:
LPS-63617 Apply new method of UpgradeTable

dantewang committed:
LPS-63617 More LoggingTimer, com.liferay.portal.upgrade.v6_0_0

dantewang committed:
LPS-63617 More LoggingTimer, com.liferay.portal.upgrade.v5_2_8_to_6_0_5

dantewang committed:
LPS-63617 More LoggingTimer, com.liferay.portal.upgrade.v6_1_1

dantewang committed:
LPS-63617 More LoggingTimer, com.liferay.portal.upgrade.v6_1_0

dantewang committed:
LPS-63617 More LoggingTimer, com.liferay.portal.upgrade.v6_0_5

dantewang committed:
LPS-63617 More LoggingTimer, com.liferay.portal.upgrade.v6_0_3

dantewang committed:
LPS-63617 More LoggingTimer, com.liferay.portal.upgrade.v6_0_2

tinatian committed:
LPS-63617 SF, remove unuseful method since we have added the one with variable arguments

tinatian committed:
LPS-63617 SF, com.liferay.portal.upgrade.v6_0_0

tinatian committed:
LPS-63617 SF, com.liferay.portal.upgrade.v6_1_0

tinatian committed:
LPS-63617 SF,  com.liferay.portal.upgrade.v6_0_3

tinatian committed:
LPS-63617 SF, com.liferay.portal.upgrade.v5_2_8_to_6_0_5

Commit:
LRQA-21076 Update locator

linolaoj committed:
LPS-64038 - avoid freemarker error

Commit:
LPS-63855 Avoid downcasting

In biz.aQute.bndlib-3.1.0-sources.jar!/aQute/bnd/osgi/FileResource.java:75 you can see that the resources put inside `Jar` type are always `FileResource`.

kenjiheigel committed:
LRQA-23081 Additional updates

kenjiheigel committed:
LRQA-22029 Update locator for device preview

Commit:
LPS-63977 - Source formatting

pei-jung committed:
LPS-59671 Implement card view for user groups

brianchandotcom committed:
dynamic-data-mapping-io 1.0.0 apply

brianchandotcom committed:
dynamic-data-mapping-taglib 1.0.0 apply

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-taglib 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-test-util 1.0.0 prep next

brianchandotcom committed:
dynamic-data-mapping-test-util 1.0.0 apply

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-captcha 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-checkbox 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-date 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-editor 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-key-value 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-options 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-paragraph 1.0.0 prep next

pei-jung committed:
LPS-59671 Make name column text bold

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-radio 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-select 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-text 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-validation 1.0.0 prep next

evannagayama committed:
LRQA-23030 Remove unecessary page additions

brianchandotcom committed:
dynamic-data-mapping-validator 1.0.0 apply

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-validator 1.0.0 prep next

brianchandotcom committed:
artifact:ignore polls-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-rules-engine-sample-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-rules-engine-wiring 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-api 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-definition-impl 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-runtime-api 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-runtime-impl 1.0.0 prep next

brianchandotcom committed:
artifact:ignore ip-geocoder-sample-web 1.0.1 prep next

brianchandotcom committed:
artifact:ignore map-openstreetmap 1.0.0 prep next

brianchandotcom committed:
artifact:ignore map-taglib-freemarker-contributor 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-cache 1.0.0 prep next

evannagayama committed:
LRQA-23030 Update assertion for private page before adding a new page

brianchandotcom committed:
artifact:ignore portal-cache-ehcache 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-cache-memory 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-cache-single 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-configuration-cluster 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-configuration-metatype-definitions-annotations 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-configuration-metatype-definitions-equinox 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-configuration-module-configuration 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-portlet-tracker 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-servlet-context-helper 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-servlet-jsp-compiler 1.0.10 prep next

brianchandotcom committed:
portal-osgi-web-servlet-jsp-compiler 1.0.10 apply

brianchandotcom committed:
artifact:ignore portal-osgi-web-wab-extender 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-remote-rest-extender 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-remote-soap-extender 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-scheduler-single 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-scripting-beanshell 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-scripting-groovy 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-scripting-javascript 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-scripting-python 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-scripting-ruby 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-security-audit-wiring 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-security-auth-verifier 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-security-auto-login 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-security-ldap 1.0.0 prep next

brianchandotcom committed:
portal-security-ldap 1.0.0 apply

brianchandotcom committed:
artifact:ignore portal-security-service-access-policy-service 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-store-jcr 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-spring-extender 1.0.5 prep next

brianchandotcom committed:
portal-spring-extender 1.0.5 apply

brianchandotcom committed:
artifact:ignore portal-upgrade 1.0.0 prep next

brianchandotcom committed:
portal-upgrade 1.0.0 apply

brianchandotcom committed:
artifact:ignore portal-verify-extender 1.0.0 prep next

brianchandotcom committed:
artifact:ignore application-list-taglib 1.0.0 prep next

brianchandotcom committed:
application-list-taglib 1.0.0 apply

brianchandotcom committed:
artifact:ignore application-list-user-personal-site-permissions 1.0.0 prep next

brianchandotcom committed:
artifact:ignore asset-taglib-freemarker-contributor 1.0.0 prep next

brianchandotcom committed:
artifact:ignore journal-api 1.0.0 prep next

brianchandotcom committed:
journal-api 1.0.0 apply

brianchandotcom committed:
artifact:ignore journal-editor-config 1.0.0 prep next

brianchandotcom committed:
artifact:ignore journal-item-selector-api 1.0.0 prep next

brianchandotcom committed:
artifact:ignore journal-ratings-definition 1.0.0 prep next

brianchandotcom committed:
artifact:ignore journal-terms-of-use 1.0.0 prep next

brianchandotcom committed:
artifact:ignore layout-taglib-freemarker-contributor 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-control-menu-theme-contributor 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-control-menu-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-theme-contributor 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-simulation 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-simulation-device 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-taglib 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-taglib-freemarker-contributor 1.0.0 prep next

brianchandotcom committed:
artifact:ignore site-navigation-taglib-freemarker-contributor 1.0.0 prep next

Commit:
LPS-64052 Breaking changes

brianchandotcom committed:
artifact:ignore site-item-selector-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore staging-taglib 1.0.0 prep next

brianchandotcom committed:
artifact:ignore trash-taglib-freemarker-contributor 1.0.0 prep next

brianchandotcom committed:
artifact:ignore journal-content-asset-addon-entry-common 1.0.0 prep next

brianchandotcom committed:
artifact:ignore journal-content-asset-addon-entry-conversions 1.0.0 prep next

marcellustavares committed:
LPS-64038 SF

brianchandotcom committed:
artifact:ignore journal-content-asset-addon-entry-print 1.0.0 prep next

Commit:
LPS-63805 I broke this when I tried to improve the performance, sorry. Pass the proper value to the prepared statment

brianchandotcom committed:
artifact:ignore journal-content-asset-addon-entry-ratings 1.0.0 prep next

brianchandotcom committed:
artifact:ignore journal-content-asset-addon-entry-related-assets 1.0.0 prep next

brianchandotcom committed:
com.liferay.application.list.taglib cache

Commit:
LPS-63805 We don't need to execute 2 queries (performance should improve)

Commit:
LPS-63805 Refactor the prepared statment variable since we've removed the previous one

brianchandotcom committed:
build.gradle auto sort

brianchandotcom committed:
com.liferay/com.liferay.application.list.taglib cache

Commit:
LPS-64130 IDE Compilation issue in Wab Extender

brianchandotcom committed:
LPS-59671 avoid chaining

brianchandotcom committed:
LPS-59671 Hugo SF

brianchandotcom committed:
LPS-59671 SF

brianchandotcom committed:
LPS-59671 Hugo SF

brianchandotcom committed:
LPS-59671 Hugo SF

brianchandotcom committed:
LPS-59671 Hugo SF

brianchandotcom committed:
LPS-59671 match modules/apps/foundation/user-groups-admin/user-groups-admin-web/src/main/resources/META-INF/resources/edit_user_group_assignments.jsp

brianchandotcom committed:
LPS-63855 SF

brianchandotcom committed:
LPS-59671 SF

kenjiheigel committed:
LRQA-23095 Check property to determine if it's using the PR tester or not

brianchandotcom committed:
LPS-63855 SF

brianchandotcom committed:
LPS-63855 SF

brianchandotcom committed:
artifact:ignore marketplace-service 1.0.0 prep next

brianchandotcom committed:
artifact:ignore document-library-google-docs 1.0.0 prep next

brianchandotcom committed:
artifact:ignore flags-service 1.0.0 prep next

brianchandotcom committed:
artifact:ignore flags-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore invitation-invite-members-service 1.0.0 prep next

brianchandotcom committed:
artifact:ignore item-selector-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore microblogs-service 1.0.0 prep next

brianchandotcom committed:
artifact:ignore microblogs-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore quick-note-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore ratings-page-ratings-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore recent-documents-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore social-activity-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore social-group-statistics-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore social-networking-service 1.0.1 prep next

brianchandotcom committed:
artifact:ignore social-user-statistics-web 1.0.0 prep next

brianchandotcom committed:
social-user-statistics-web 1.0.0 apply

brianchandotcom committed:
artifact:ignore wiki-navigation-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-service 1.0.0 prep next

brianchandotcom committed:
dynamic-data-mapping-service 1.0.0 apply

brianchandotcom committed:
artifact:ignore polls-web 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-runtime-impl 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-service 1.0.0 prep next

brianchandotcom committed:
artifact:ignore contacts-api 1.0.2 prep next

brianchandotcom committed:
artifact:ignore contacts-service 1.0.0 prep next

brianchandotcom committed:
artifact:ignore hello-velocity-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore hello-world-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore license-manager-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-background-task-service 1.0.0 prep next

brianchandotcom committed:
portal-background-task-service 1.0.0 apply

brianchandotcom committed:
artifact:ignore portal-lock-service 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-servlet-jsp-compiler 1.0.11 prep next

brianchandotcom committed:
portal-osgi-web-servlet-jsp-compiler 1.0.11 apply

brianchandotcom committed:
artifact:ignore portal-osgi-web-wab-extender 1.0.1 prep next

brianchandotcom committed:
artifact:ignore portal-scheduler-quartz 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-security-service-access-policy-service 1.0.1 prep next

brianchandotcom committed:
artifact:ignore asset-entry-query-processor-custom-user-attributes 1.0.0 prep next

brianchandotcom committed:
artifact:ignore asset-tags-compiler-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore asset-tags-navigation-web 1.0.0 prep next

brianchandotcom committed:
asset-tags-navigation-web 1.0.0 apply

brianchandotcom committed:
artifact:ignore nested-portlets-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portlet-configuration-css-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore portlet-configuration-sharing-web 1.0.0 prep next

brianchandotcom committed:
LPS-59671 fix

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-simulation-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore product-navigation-site-administration 1.0.0 prep next

brianchandotcom committed:
product-navigation-site-administration 1.0.0 apply

brianchandotcom committed:
artifact:ignore product-navigation-user 1.0.0 prep next

brianchandotcom committed:
artifact:ignore rss-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore site-navigation-breadcrumb-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore site-navigation-directory-web 1.0.0 prep next

brianchandotcom committed:
LPS-64052 NetBeans and Eclipse

brianchandotcom committed:
artifact:ignore site-navigation-language-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore site-navigation-menu-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore site-navigation-site-map-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore site-browser-web 1.0.0 prep next

brianchandotcom committed:
LPS-64052 not used

brianchandotcom committed:
artifact:ignore xsl-content-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore layout-type-controller-control-panel 1.0.0 prep next

brianchandotcom committed:
artifact:ignore layout-type-controller-full-page-application 1.0.0 prep next

brianchandotcom committed:
artifact:ignore layout-type-controller-node 1.0.0 prep next

brianchandotcom committed:
artifact:ignore layout-type-controller-shared-portlet 1.0.0 prep next

hhuijser committed:
LPS-61420 Catch statements that are confusing to read

hhuijser committed:
LPS-61420 Create new var for better readability

brianchandotcom committed:
LPS-63617 SF

brianchandotcom committed:
LPS-63617 SF

brianchandotcom committed:
LPS-64052 SF

patriciajeanperez committed:
LRQA-21571 Click on Organization Nav Bar

brianchandotcom committed:
build.gradle auto sort

patriciajeanperez committed:
LRQA-21571 Naviagte back to Users and Organizations

patriciajeanperez committed:
LRQA-21571 Update macro to match UI in adding categories

patriciajeanperez committed:
LRQA-21571 Use correct locators

brianchandotcom committed:
LPS-59671 fix compile

brianchandotcom committed:
com.liferay.asset.tags.navigation.web cache

patriciajeanperez committed:
LRQA-21571 SF

brianchandotcom committed:
com.liferay.product.navigation.site.administration cache

brianchandotcom committed:
LPS-61420 prep next, use latest

brianchandotcom committed:
LPS-61420 prep next, use latest

brianchandotcom committed:
LPS-61420 com.liferay.source.formatter 1.0.160 cache

brianchandotcom committed:
Record

sharonchoi commented:
SF for PPerez:
-please move macros back into testcase, per confirmation from BLee and send directly to him.
@patriciajeanperez
